exercises/protein-translation/canonical-data.json

```diff
@@ -0,0 +1,143 @@
+        "Returns the name of the protein if given RNA is valid, "
+      , "else throws an error. "
+      ],
+      "cases": [
+        {
+          "description": "Methionine RNA sequence",
+          "property": "translate",
+          "strand": "AUG",
+          "expected": ["Methionine"]
+        },
+        {
+          "description": "Phenylalanine RNA sequence 1",
+          "property": "translate",
+          "strand": "UUU",
+          "expected": ["Phenylalanine"]
+        },
+        {
+          "description": "Phenylalanine RNA sequence 2",
+          "property": "translate",
+          "strand": "UUC",
+          "expected": ["Phenylalanine"]
+        },
+        {
+          "description": "Leucine RNA sequence 1",
+          "property": "translate",
+          "strand": "UUA",
+          "expected": ["Leucine"]
+        },
+        {
+          "description": "Leucine RNA sequence 2",
+          "property": "translate",
+          "strand": "UUG",
+          "expected": ["Leucine"]
+        },
+        {
+          "description": "Serine RNA sequence 1",
+          "property": "translate",
+          "strand": "UCU",
+          "expected": ["Serine"]
+        },
+        {
+          "description": "Serine RNA sequence 2",
+          "property": "translate",
+          "strand": "UCC",
+          "expected": ["Serine"]
+        },
+        {
+          "description": "Serine RNA sequence 3",
+          "property": "translate",
+          "strand": "UCA",
+          "expected": ["Serine"]
+        },
+        {
+          "description": "Serine RNA sequence 4",
+          "property": "translate",
+          "strand": "UCG",
+          "expected": ["Serine"]
+        },
+        {
+          "description": "Tyrosine RNA sequence 1",
+          "property": "translate",
+          "strand": "UAU",
+          "expected": ["Tyrosine"]
+        },
+        {
+          "description": "Tyrosine RNA sequence 2",
+          "property": "translate",
+          "strand": "UAC",
+          "expected": ["Tyrosine"]
+        },
+        {
+          "description": "Cysteine RNA sequence 1",
+          "property": "translate",
+          "strand": "UGU",
+          "expected": ["Cysteine"]
+        },
+        {
+          "description": "Cysteine RNA sequence 2",
+          "property": "translate",
+          "strand": "UGC",
+          "expected": ["Cysteine"]
+        },
+        {
+          "description": "Tryptophan RNA sequence",
+          "property": "translate",
+          "strand": "UGG",
+          "expected": ["Tryptophan"]
+        },
+        {
+          "description": "STOP codon RNA sequence 1",
+          "property": "translate",
+          "strand": "UAA",
+          "expected": []
+        },
+        {
+          "description": "STOP codon RNA sequence 2",
+          "property": "translate",
+          "strand": "UAG",
+          "expected": []
+        },
+        {
+          "description": "STOP codon RNA sequence 3",
```

Comment:
I'd like to see tests for more of the simple STOP codon sequences.
`UAGUGG`
`UGGUAG`
`UGGUAGUGG`